.github/workflows/pr-open.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     uses: ./.github/workflows/.deploy.yml
     with:
       db_testdata: true
-      triggers: ('db/' 'libs/' 'api/' 'admin/' 'public/')
+      # triggers: ('db/' 'libs/' 'api/' 'admin/' 'public/')
```

Comment:
Why is this commented out?

Reply:
That was just me triggering a PR deploy.  Removed!